// This is the solution file.
// Scroll to the bottom, but there are spoilers!!
















































// 🐽 almost there


















































/*

return new Promise((resolve, reject) => {

  resolve('🐊'); // 🤩🤩🤩 THIS LINE

}).then((maybeSnack) => {

  // the first function in `then` is called when
  // the previous chain is `resolve`d
  // ⭐ Getting here is the goal

  assert(maybeSnack === '🐊', 'Omnomnom ty --Princess');
}).catch((maybeSnack) => {

  // the first function in `catch` is called when
  // the previous chain is `reject`d or an error is thrown

  assert(false, '😩 How do we get to the `then` above?');
});

*/
